library/lib/serve.js

```diff
@@ -33,14 +33,20 @@ const envRequire = isProduction ? require : require('import-fresh')
 
 const notCached = ['html', 'txt', 'json', 'xml']
 
+const { contentSecurityPolicy, ...helmetOptionsToUse } = helmetOptions || {}
+const cspMiddleware = contentSecurityPolicy && createCspMiddleware(contentSecurityPolicy)
+const [sendHtmlFunction, sendHtmlFunctionIsAsync] = cspMiddleware
+  ? [sendHtmlWithCspHeaders, true]
+  : [sendHtml, false]
+
 if (isProduction) app.use(morgan('combined'))
 app.use(helmet(Object.assign(
   {
     hsts: false, // hsts-headers are sent by our loadbalancer
     contentSecurityPolicy: false,
```

Comment:
It feels a bit strange that we have 
```
const { contentSecurityPolicy, ...helmetOptionsToUse } = helmetOptions || {}
```
and 
```
 Object.assign( {
    hsts: false, // hsts-headers are sent by our loadbalancer
    contentSecurityPolicy: false,
    referrerPolicy: { policy: 'strict-origin-when-cross-origin' },
  },helmetOptionsToUse)
```
Does that mean that the resulting object has always `contentSecurityPolicy: false`?